flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/StreamSourceOperatorLatencyMetricsTest.java

```diff
@@ -163,7 +164,7 @@ private void testLatencyMarkEmission(int numberLatencyMarkers, OperatorSetupOper
 		operatorSetup.setupSourceOperator(operator, testProcessingTimeService);
 
 		// run and wait to be stopped
-		operator.run(new Object(), mock(StreamStatusMaintainer.class), new CollectorOutput<Long>(output));
+		operator.run(new Object(), mock(StreamStatusMaintainer.class), new CollectorOutput<Long>(output), mock(OperatorChain.class));
```

Comment:
Please do not introduce new mockito mocked classes. As usually either:
- please instantiate a real `OpeartorChain`
- please instantiate a real `OpeartorChain` with maybe mocked content `RecordWriters `
- implement `OperatorChain` mock, maybe a `NoOpOperatorChain`

Keep in mind that `mocked` I mean a proper mocks, not mockito.

(You can read about the reasoning behind not using mockito in the proposed code style guideline)